src/test/rustdoc-gui/huge-logo.goml

```diff
@@ -0,0 +1,19 @@
+goto: "file://" + |DOC_PATH| + "/huge_logo/index.html"
+
+size: (1280, 1024)
+// offsetWidth = width of sidebar
+assert-property: (".sidebar .logo-container", {"offsetWidth": "200", "offsetHeight": 100})
+assert-property: (".sidebar .logo-container img", {"offsetWidth": "100", "offsetHeight": 100})
```

Comment:
So max image size is 100px (please tell me if I misunderstood). If I understood it correctly, sounds good to me! :+1: